xpano/gui/panels/about.cc

```diff
@@ -74,18 +82,26 @@ void AboutPane::Draw() {
   ImGui::Begin("About", &show_, window_flags);
 
   ImGui::SetNextItemWidth(ImGui::GetContentRegionAvail().x);
+
+  if(license_text[1] == DefaultNotice() && IsReady(licenses_)) {
+    auto temp = licenses_.get();
+    license_text.clear();
```

Comment:
instead of the clear, copy the licenses over using std::copy, example [Code location](https://github.com/krupkat/xpano/blob/d5c968bb315df060d0e817266d353f6b538e7650/xpano/log/logger.cc#L93)